tests/ut/data/scargo_test_esp32.toml

```diff
@@ -6,7 +6,7 @@ homepage-url = "www.hello-world.com"
 
 bin_name = "esp32proj"
 target = "esp32"
-build-env = "docker"
+build-env = "native"
```

Comment:
please keep docker as default

Reply:
The reason why I changed it to _native_ is that these tomls are used in tests only and I don't want to trigger scargo to run again same command in docker.